src/graph/OrderByExecutor.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef GRAPH_ORDERBYEXECUTOR_H_
+#define GRAPH_ORDERBYEXECUTOR_H_
+
+#include "base/Base.h"
+#include "graph/TraverseExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+class OrderByExecutor final : public TraverseExecutor {
+public:
+    OrderByExecutor(Sentence *sentence, ExecutionContext *ectx);
+
+    const char* name() const override {
+        return "OrderByExecutor";
+    }
+
+    Status MUST_USE_RESULT prepare() override;
+
+    void execute() override;
+
+    void feedResult(std::unique_ptr<InterimResult> result) override;
+
+    void setupResponse(cpp2::ExecutionResponse &resp) override;
+
+private:
+    std::unique_ptr<InterimResult> setupInterimResult();
+
+private:
+    OrderBySentence                                            *sentence_{nullptr};
+    std::unique_ptr<InterimResult>                              inputs_;
+    std::vector<cpp2::RowValue>                                 rows_;
+    std::vector<std::pair<int64_t, OrderFactor::OrderType>>     sortFactors_;
```

Comment:
Explain uint64_t is the field index of schema

Reply:
check SchemaProvider::getFieldIndex, it returns integer